GuideFinder-Mac/GuideFinder_Mac_2020_SMM_PAMless.Rmd

```diff
@@ -0,0 +1,1365 @@
+  NewCoordinatesDF<-rbind(NewCoordinatesDF,Temp)
+} }
+names(NewCoordinatesDF)<-c("geneproduct","strand", "start", "end")  #change column names to those that are used by the rest of the Guide Finder script
+
+```
+
+
+##6.Set Parameters for Guide Design
+Change as desired. 
+Set the following:
+
+ *__N__: Set minimum guide length (recommended 17-23); set as an integer 
+* __GC_Min__ : GC minimum; set as a decimal (e.g. for GC min of 35%,  set to .35)
+* __DistanceAsPercentageofCDS__ : Maximum distance of guide from the transcription start site 
+    + As percentage of gene. For example, setting to .5 tell the program to look in the first 50% of the gene for guides
+* __GuidesPerGene__ : Maximum number of guides/gene the program should output 
+* __DistbtGuides__ : Minimum distance between guides
+    + For paired guide design. For example, using two guides to knockdown one gene in the same cell. In these cases, guides should be an appropriate distance apart. Overlapping guides have been shown to reduce extent of knockdown. 
+    + If only designing guides for single-guide knockdown, set DistbtGuides to 0 
+ * __Test_bad_seed__ : Set to TRUE or FALSE, if you want to turn this function on or off
+ *__Bad_seed__: If you want to remove sequences with a "bad seed", add the sequences here, in quotations 
+ *__Test_RE__: Set to TRUE or FALSE, if you want to turn this function on or off
+ *__RE_site__ If you want to remove sequences with a "bad seed", add the sequences here
+ *__PAMless_guides__: Set to TRUE or FALSE, if you want to include guides made from any PAM site (FALSE will search for NGG guides only)
+
+```{r,tidy=TRUE}
+
+N <- 20 
+GC_Min<-.35
+DistanceAsPercentageofCDS<-.50
+GuidesPerGene<-10
+DistbtGuides<-100
+Test_bad_seed <- TRUE 
+Bad_seed <- "ACCCA"  
+Test_RE <- FALSE 
+RE_site <- "GGTCTC" 
+PAMless_guides <- TRUE
+
+```
+
+
+##7.Manual Guide Selection  
+*Complete genomes only* If the user wants to run the program on a subset of genes from a draft genome, select these genes during the pre-processing script. Locus tags/unique identifiers differ between draft genome files.   
+
+* If creating guides for one gene or a subset of genes, set Manual Selection to TRUE. If whole-genome, set to FALSE
+* If Manual Selection is TRUE, add locus_tags (unique gene identifiers). 
+    + Can manually add any number desired. __Example format__ : Replace bolded information with the unique gene identifiers for 
+    each of the genes you want to select. GenesToSelect<-c( __"SMU_01"__, __"SMU_02"__, __"SMU_05"__)
+
+```{r,tidy=TRUE}
+GenesToSelect<-c("NEB5A_00995", "NEB5A_00940", "NEB5A_19010", "NEB5A_16190", "NEB5A_20195", "NEB5A_04095", "NEB5A_17070")
+ManualSelection<-TRUE
+```
+
+***
+## __OPTIONAL: Lower thresholds to find guides for genes that did not produce any with primary thresholds__  
```

Comment:
Again, maybe a quick note of how this plays with PAMless? Is is less likely we would want to run this option since PAMless produces so many guides?